apache-activemq-mixin/dashboards/apache-activemq-instance-overview.libsonnet

```diff
@@ -0,0 +1,1086 @@
+    justifyMode: 'auto',
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'auto',
+  },
+  pluginVersion: '10.2.0-59981',
+};
+
+local queueSizePanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(instance, activemq_cluster) (activemq_queue_queue_size{instance=~"$instance", activemq_cluster=~"$activemq_cluster", job=~"$job"})',
+      datasource=promDatasource,
+      legendFormat='{{activemq_cluster}} - {{instance}} - queue',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Queue size',
+  description: 'Number of messages on queue destinations, including any that have been dispatched but not acknowledged.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 25,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'smooth',
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'normal',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local destinationMemoryUsagePanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(instance, activemq_cluster) (activemq_queue_memory_percent_usage{job=~"$job", activemq_cluster=~"$activemq_cluster", instance=~"$instance"})',
+      datasource=promDatasource,
+      legendFormat='{{activemq_cluster}} - {{instance}} - queue',
+    ),
+    prometheus.target(
+      'sum by(instance, activemq_cluster) (activemq_topic_memory_percent_usage{job=~"$job", activemq_cluster=~"$activemq_cluster", instance=~"$instance"})',
+      datasource=promDatasource,
+      legendFormat='{{activemq_cluster}} - {{instance}} - topic',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Destination memory usage',
+  description: 'The percentage of memory being used by topic and queue destinations.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'continuous-BlYlRd',
```

Comment:
<img width="663" alt="image" src="https://github.com/grafana/jsonnet-libs/assets/38274348/ed9433fd-f87d-4f13-b30c-9cbb2b41a8ef">
I did it following this style guideline bullet point. Since this is a percentage timeseries panel